.gitignore

```diff
@@ -24,4 +24,3 @@ MANIFEST
 .eggs/
 *.so
 *~
-.pytest_cache
```

Comment:
This is being removed from `.gitignore`? Shouldn't it be there? I have something similar for other projects using pytest.

Reply:
That was me screwing up the rebase, I've dropped that change (so `.pytest_cache` is still being ignored).